daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -1579,13 +1585,19 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
       case (RefQName(_, "logicalTypeValueInt", DFDL), args) =>
         FNZeroArgExpr(functionQNameString, functionQName,
           NodeInfo.Integer, NodeInfo.AnyAtomic, DFDLXLogicalTypeValueInt(_, _))
-          
+
       case (RefQName(_, "logicalTypeValueString", DFDL), args) =>
         FNZeroArgExpr(functionQNameString, functionQName,
           NodeInfo.String, NodeInfo.AnyAtomic, DFDLXLogicalTypeValueString(_, _))
 
       //End typeValueCalc related functions
 
+      case (RefQName(_, "lookAhead", DAF_APACHE), args) =>
+        FNTwoArgsExpr(functionQNameString, functionQName, args,
+          NodeInfo.NonNegativeInteger, NodeInfo.UnsignedInt, NodeInfo.UnsignedInt, DFDLXLookAhead(_))
```

Comment:
I don't see DFDLXLookAhead defined anywhere. Did you forget to add a file?

Reply:
Fixed. I also took the opportunity to rename in to DAFLookAhead.

@mbeckerle  @jadams-tresys  You might want to re-review since I was missing the guts of the PR.